stdlib/sys.mli

```diff
@@ -419,13 +417,12 @@ type extra_prefix = Plus | Tilde
 type extra_info = extra_prefix * string
```

Comment:
this needs to wait until https://github.com/ocaml-flambda/flambda-backend/pull/3664 also merges